.golangci.yml

```diff
@@ -19,8 +19,7 @@ linters-settings:
     replace-allow-list:
       # replace containerd GHSA-5ffw-gxpp-mxpf
       - github.com/containerd/containerd
-      # trivy requires these replacements
-      - oras.land/oras-go
+      - github.com/docker/docker
```

Comment:
please add a comment for the reason of the replace

Reply:
Added